pyproject.toml

```
@@ -50,7 +51,7 @@ requests-mock = "^1.9.3"
 [tool.poetry.group.dev.dependencies]
 coverage = { version = "^7.0.5", extras = ["toml"] }
 flake8 = "^6.0.0"
-pre-commit = "^2.6.0"
+pre-commit = "^3.0.0"
 black = { version = ">=22.10.0", extras = ["jupyter"] }
```

```suggestion
black = { version = ">=23.1.0", extras = ["jupyter"] }
```
To keep it in sync with the updated version in `.pre-commit-config.yaml`